integration-tests/mock-components/updated-promise.js

```diff
@@ -0,0 +1,12 @@
+const { registerHtml, useObservable, useEffect } = require('../../src/tram-one')
+const html = registerHtml()
+
+module.exports = ({ initialCount }) => {
+	const [counter] = useObservable({ count: Number(initialCount) })
+	const increment = () => { counter.count += 1 }
+	useEffect(async () => {
+		document.title = `The count is ${counter.count}`
+	})
```

Comment:
yes! `useEffect` supports async... well, it should have, now it does for sure-sure.